recipe/meta.yaml

```diff
@@ -42,8 +48,8 @@ about:
     effectiveness of tests. It can show which parts of your code are being
     exercised by tests, and which are not.
   doc_url: http://coverage.readthedocs.org
```

Comment:
URLs should prefer HTTPS over HTTP.

Reply:
The URL was corrected:
https://github.com/AnacondaRecipes/coverage-feedstock/blob/update_to_6.1.1/recipe/meta.yaml#L50&L52